tests/test_fuzzy_checker.py

```diff
@@ -0,0 +1,4 @@
+# TODO: Add tests for fuzzy_checker
+
+def test_placeholder():
```

Comment:
nit: add a skip marker w/ a TODO description so we actually see this when running (verbose) tests